torch/_C/__init__.pyi.in

```diff
@@ -602,7 +602,9 @@ class Node:
     def t_(self, name: str, val: Tensor) -> Node: ...
     def ts(self, name: str) -> List[Tensor]: ...
     def ts_(self, name: str, val: List[Tensor]) -> Node: ...
+    def ty(self, name: str) -> JitType: ...
     def ty_(self, name: str, val: JitType) -> Node: ...
+    def tys(self, name: str) -> List[JitType]: ...
```

Comment:
Curious about the reason of adding these two here

Reply:
These were missing. It was discovered due to that the torch script version of `RoiAlign` and `RoiPool` used it. Although later it turned out we could not support the torch script version due to `prim::isinstance`.